File: project/2.07_その他のラップクラス/java/java_ex_2_07_2/src/java_ex_2_07_2/Ex_2_07_2.java
package java_ex_2_07_2;

public class Ex_2_07_2 {

	public static void main(String[] args) {

		// 16進数
		short va_16 = Short.parseShort("22",16);
		System.out.println( va_16 );
		
		// 8進数
		short va_8 = Short.parseShort("33",8);
		System.out.println( va_8 );
		
		// 10進数
		short va_10 = Short.parseShort("44",10);
		System.out.println( va_10 );
		
		// 合計値
		System.out.println( va_16 + va_8 + va_10 );
	}

}

File: project/2.07_その他のラップクラス/py/py_ex_2_07_2/src/ex_2_07_2.py
'''
Created on 2023/03/23

@author: Keroichi.T
'''

def main():
    # 16進数を10進数に
    va_16:int = int( "22", 16 )
    print( va_16 )

    # 16進数を8進数に
    va_8:int = int( "33", 8 )
    print( va_8 )

    # 16進数を10進数に
    va_10:int = int( "44", 10 )
    print( va_10 )

    # 合計値
    print( va_16 + va_8 + va_10 )

if __name__ == '__main__':
    main()